apis/python/README-dev.md

```diff
@@ -2,7 +2,6 @@
 * Pre-push suggestions:
   * `black . tools/[a-z]*`
   * `isort . tools/[a-z]*`
-  * `flake8 . tools/[a-z]*`
```

Comment:
Why not say `ruff` here too?

Reply:
Same reason as for setup.py; I added `ruff` for now [here](https://github.com/single-cell-data/TileDB-SOMA/pull/751/commits/5c7a4fd90b95de624d60434bd27b3758639a350f).